ui/assets/js/common.test.js

```diff
@@ -1,4 +1,4 @@
-import { EscapeRegExp, ExpandVars, UrlToRepo } from "./common";
+import { EscapeRegExp, ExpandVars, UrlToRepo, UrlParts } from "./common";
```

Comment:
thank you! I'm surprised tests were passing before this import was added 0_o

Reply:
Same here, looks like npm tests are not set-up in the [workflow](https://github.com/hound-search/hound/blob/main/.github/workflows/go.yaml#L81)